### DISCUSSION TITLE
Make sure 2FA is enabled in your GitHub account

### DISCUSSION THREAD

**Discussion by gastaldi** [quarkiverse/quarkiverse#101] [Team Posts]
As a security measure, starting on December 1st, 2022, Quarkiverse Hub members are required to [enable two-factor authentication in their GitHub accounts](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication).

Let me know if you need any help setting that up. After the beforementioned date, users that do not have 2FA enabled will have their access revoked from the organization.

**Comment by ppalaga** [comment]
How can I check that I comply with your requirements?

**Comment by gastaldi** [comment]
Check if 2FA is enabled in https://github.com/settings/security

**Comment by gastaldi** [comment]
ACTION REQUIRED: Friendly reminder that 2FA is going to be required next week. If you haven't yet enabled 2FA in your account please do so. Ping me if you need help setting it up

**Comment by gastaldi** [comment]
The following people haven't enabled 2FA yet and will have their access revoked on the Quarkiverse org on Dec 1st, 2022. Please act now:
 
- @alexeysharandin
- @aruffie
- @bwijsmuller 
- @cameronprater 
- @click33 
- @Dufgui 
- @dufoli
- @gustavocoding
- @marcinczeczko
- @naah69
- @oscarfh
- @oztimpower
- @shumonsharif
- @Spanjer1 
- @vietk
- @zohar-soul

**Comment by shumonsharif** [comment]
It appears I never got any [notifications](https://github.com/notifications) for the previous messages and comments on this topic, until you actually tagged me @gastaldi .  I tracked down the issue to the fact that my quarkiverse-members 'Watch' settings only notify if I'm participating or tagged in a discussion.  I assume this is the default setting, as I never touched it.

![image](https://user-images.githubusercontent.com/13334073/203664115-a25cca00-aa2c-4b82-a87e-a6c73c0b4061.png)

Not sure if there's some way to fix this - ie. notifications should go out by default to all `quarkiverse-members` members.

**Comment by gastaldi** [comment]
Thanks for letting us know @shumonsharif ! 

I haven't found a way to fix this, but I think we can somehow trigger updates when mentioning the `@quarkiverse/quarkiverse-members` team directly

**Comment by gastaldi** [comment]
Only two days to go. The following people still haven't enabled 2FA. Please do so by tomorrow to avoid having your account privileges revoked:

- @aruffie
- @bwijsmuller
- @cameronprater
- @click33
- @Dufgui
- @dufoli
- @gustavocoding
- @marcinczeczko
- @oscarfh
- @oztimpower

**Comment by gastaldi** [comment]
Starting today,  [Quarkiverse Hub](https://github.com/quarkiverse) members are required to [enable two-factor authentication in their GitHub accounts](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication). 

The following users will have their access revoked from the following repositories, which will be re-granted access once they enable 2FA in their accounts: 

@dufoli
  - https://github.com/quarkiverse/quarkus-cxf
  - https://github.com/quarkiverse/quarkus-jna
  - https://github.com/quarkiverse/quarkus-rsocket

@Dufgui
  - https://github.com/quarkiverse/quarkus-cxf

@gustavocoding
  - https://github.com/quarkiverse/quarkus-lucene

@oztimpower
  - https://github.com/quarkiverse/quarkus-amazon-alexa

@click33
  - https://github.com/quarkiverse/quarkus-sa-token

@cameronprater
  - https://github.com/quarkiverse/quarkus-discord4j

@aruffie
  - https://github.com/quarkiverse/quarkus-univocity-parsers
 
Best Regards.
